documentation/modules/cruise-control/con-cruise-control-overview.adoc

```diff
@@ -2,54 +2,501 @@
+
+
+[[contents-optimization-proposals]]
+=== The results of an optimization proposal
+
+When an optimization proposal is generated, a summary and broker load is returned.
+
+Summary:: The summary is contained in the `KafkaRebalance` resource. The summary provides an overview of the proposed cluster rebalance and indicates the scale of the changes involved.
+A summary of a successfully generated optimization proposal is contained in the `Status.optimizationResult` property of the `KafkaRebalance` resource.
+The information provided is a summary of the full optimization proposal.
+Broker load:: The broker load is stored in a ConfigMap that contains data as a JSON string. The broker load shows before and after values for the proposed rebalance, so you can see the impact on each of the brokers in the cluster.
+
+=== Manually approving or rejecting an optimization proposal
+
+An optimization proposal summary shows the proposed scope of changes.
+
+You can use the name of the `KafkaRebalance` resource to return a summary from the command line.
+
+.Returning an optimization proposal summary
+[source,shell]
+----
+kubectl describe kafkarebalance <kafka_rebalance_resource_name> -n <namespace>
+----
+
+You can also use the `jq` {JQTool}.
+
+.Returning an optimization proposal result using jq
+[source,shell]
+----
+kubectl get kafkarebalance <kafka_rebalance_resource_name> -n <namespace> -o json | jq '.status.optimizationResult'
+----
+
+Use the summary to decide whether to approve or reject an optimization proposal.
+
+Approving an optimization proposal:: You approve the optimization proposal by setting the `strimzi.io/rebalance` annotation of the `KafkaRebalance` resource to `approve`.
+Cruise Control applies the proposal to the Kafka cluster and starts a cluster rebalance operation.
+Rejecting an optimization proposal:: If you choose not to approve an optimization proposal,
+you can xref:proc-generating-optimization-proposals-str[change the optimization goals] or xref:con-rebalance-{context}[update any of the rebalance performance tuning options], and then generate another proposal.
+You can generate a new optimization proposal for a `KafkaRebalance` resource by setting the `strimzi.io/rebalance` annotation to `refresh`.
+
+Use optimization proposals to assess the movements required for a rebalance.
+For example, a summary describes inter-broker and intra-broker movements.
+Inter-broker rebalancing moves data between separate brokers.
+Intra-broker rebalancing moves data between disks on the same broker when you are using a JBOD storage configuration.
+Such information can be useful even if you don't go ahead and approve the proposal.
+
+You might reject an optimization proposal, or delay its approval, because of the additional load on a Kafka cluster when rebalancing.
```

Comment:
We might want to add that if the proposal is too old, the cluster load may have changed significantly, so it would be better to request a new proposal.